sdks/python/apache_beam/io/external/kafka.py

```diff
@@ -166,6 +171,8 @@ def __init__(
                                serialization.ByteArraySerializer'.
     :param expansion_service: The address (host:port) of the ExpansionService.
     """
+    if expansion_service is None:
```

Comment:
Prob. we don't need this since we default to default_io_expansion_service() below ?